contentpal/src/main/java/org/dmfs/android/contentpal/operations/Referring.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017 dmfs GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dmfs.android.contentpal.operations;
+
+import android.support.annotation.NonNull;
+
+import org.dmfs.android.contentpal.InsertOperation;
+import org.dmfs.android.contentpal.Operation;
+import org.dmfs.android.contentpal.RowSnapshot;
+
+
+/**
+ * A {@link InsertOperation} decorator that a foreign key to another row of type {@code <V>}.
```

Comment:
Please also fix this Javadoc.

Reply:
Fixed, also removed the generic type of the 'foreign table' - I suppose it's not needed.